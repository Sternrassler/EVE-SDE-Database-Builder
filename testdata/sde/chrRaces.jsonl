{"raceID":1,"raceName":"Caldari","description":"The Caldari State is a corporate dictatorship.","iconID":1441,"shipTypeID":null}
{"raceID":2,"raceName":"Minmatar","description":"The Minmatar Republic is a tribal society.","iconID":1442,"shipTypeID":null}
{"raceID":4,"raceName":"Amarr","description":"The Amarr Empire is a theocratic empire.","iconID":1443,"shipTypeID":null}
