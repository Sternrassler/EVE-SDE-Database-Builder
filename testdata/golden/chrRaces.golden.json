[
  {
    "raceID": 1,
    "raceName": "Caldari",
    "description": "The Caldari State is a corporate dictatorship.",
    "iconID": 1441
  },
  {
    "raceID": 2,
    "raceName": "Minmatar",
    "description": "The Minmatar Republic is a tribal society.",
    "iconID": 1442
  },
  {
    "raceID": 4,
    "raceName": "Amarr",
    "description": "The Amarr Empire is a theocratic empire.",
    "iconID": 1443
  }
]